README.md

```diff
@@ -14,9 +14,9 @@ Why?
 Because we are too lazy for manually setting up local environments for each project we have to work on.  
 In short, we need to:
 
-- Work on any new/existing project from any platform in minuts
+- Be able to work on any new/existing project from any platform in minuts
+- Enable/disable any package or language support as well
```

Comment:
I preferred the previous version :/ (`Enable/disable support for languages, packages or any various utility as well `)